docs/python-examples/room-assignment-to-walls.py

```
@@ -0,0 +1,83 @@
+Input: Room assignment data from Analysis Node (with room_assignment metric)
+Output: Mapping of wall GlobalIds to room names for Property Node
+
+Usage:
+1. Connect Analysis Node (room_assignment) → Python Node → Property Node → Export Node
+2. Set Property Node to:
+   - Action: Set
+   - Property Name: SpaceName (or RoomName)
+   - Target Pset: Pset_WallCommon
+"""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)